src/auth/auth.service.ts

```diff
@@ -30,4 +51,42 @@ export class AuthService {
 
     return userCredentials.userId;
   }
+
+  async signIn(signInRequest: SignInRequest): Promise<string> {
+    const userCredentials =
+      await this.userCredentialsRepository.getUserCredentialsByUserId(
+        signInRequest.userId,
+      );
+
+    if (!userCredentials) {
+      throw new UnauthorizedException();
+    }
+
+    return this.generateToken(
+      signInRequest.userId,
+      process.env.JWT_SECRET ? process.env.JWT_SECRET : 'Default_jwt_secret',
+      process.env.JWT_EXPIRY_TIME ? process.env.JWT_EXPIRY_TIME : '1h',
```

Comment:
you should be able to shorten these two lines using `??` operator:
```suggestion
      process.env.JWT_SECRET ?? 'Default_jwt_secret',
      process.env.JWT_EXPIRY_TIME ?? '1h',
```